### ISSUE TITLE
Network description text file

### ISSUE THREAD

**Issue by pranjaldaga** [senarvi/theanolm#16]
The [documentation](http://theanolm.readthedocs.io/en/latest/training.html#network-structure-description) says: 
**Network structure description**
The neural network layers are specified in a text file. In which text file should I specify the network layers? 
Thanks!

**Comment by senarvi** [comment]
You just have to create such file with a text editor. There's a really simple example in the documentation that creates three layers: projection layer, LSTM layer and softmax layer. You can copy & paste it to get started.

**Comment by pranjaldaga** [comment]
@senarvi Thanks! That was pretty much clear through the documentation. Just wanted to know if there is a particular name (and path) given to this file and how is this file called from theanolm!

**Comment by senarvi** [comment]
Oh, the name can be anything. You just pass the path to the file with `--architecture` command line argument.

**Comment by pranjaldaga** [comment]
Ah! That makes total sense now. Thanks!
